src/main/java/duke/logging/TaskList.java

```diff
@@ -0,0 +1,155 @@
+package duke.logging;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+import duke.exception.InvalidDescriptionException;
+import duke.model.Deadline;
+import duke.model.Event;
+import duke.model.Task;
+import duke.model.ToDo;
+
+/**
+ * A TaskList call denotes a system with a list of task and has the ability to make changes to the list.
+ */
+public class TaskList {
+    private final ArrayList<Task> tasks;
+
+    /**
+     * Construct a TaskList.
+     * @param tasks   A list of tasks.
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Print out all the tasks in the list.
+     */
+    public String list() {
+        String message = "     Here are the tasks in your list:";
```

Comment:
maybe can leave a blankline after this to group similar codes together for better readability?